plugins/uefi-capsule/efi/meson.build

```diff
@@ -31,38 +31,37 @@ if not have_gnu_efi
   error('gnu-efi support requested, but headers were not found')
 endif
 
-arch_lds = 'efi.lds'
+arch_lds = 'elf_@0@_efi.lds'.format(gnu_efi_path_arch)
+
 arch_crt = 'crt0.o'
-if efi_ldsdir == ''
-  efi_ldsdir = join_paths(efi_libdir, 'gnuefi', gnu_efi_path_arch)
-  cmd = run_command('test', '-f', join_paths(efi_ldsdir, arch_lds))
+if efi_crtdir == ''
+  efi_crtdir = join_paths(efi_libdir, 'gnuefi', gnu_efi_path_arch)
+  cmd = run_command('test', '-f', join_paths(efi_crtdir, arch_crt))
   if cmd.returncode() != 0
-    arch_lds = 'elf_@0@_efi.lds'.format(gnu_efi_path_arch)
     arch_crt = 'crt0-efi-@0@.o'.format(gnu_efi_path_arch)
-    efi_ldsdir = join_paths(efi_libdir, 'gnuefi')
-    cmd = run_command('test', '-f', join_paths(efi_ldsdir, arch_lds))
+    efi_crtdir = join_paths(efi_libdir, 'gnuefi')
+    cmd = run_command('test', '-f', join_paths(efi_crtdir, arch_crt))
   endif
   if cmd.returncode() != 0
-    efi_ldsdir = efi_libdir
-    cmd = run_command('test', '-f', join_paths(efi_ldsdir, arch_lds))
+    efi_crtdir = efi_libdir
+    cmd = run_command('test', '-f', join_paths(efi_crtdir, arch_crt))
     if cmd.returncode() != 0
-      error('Cannot find @0@'.format(arch_lds))
+      error('Cannot find @0@'.format(arch_crt))
     endif
   endif
 else
-  cmd = run_command('test', '-f', join_paths(efi_ldsdir, arch_lds))
+  cmd = run_command('test', '-f', join_paths(efi_crtdir, arch_crt))
   if cmd.returncode() != 0
-    arch_lds = 'elf_@0@_efi.lds'.format(gnu_efi_path_arch)
     arch_crt = 'crt0-efi-@0@.o'.format(gnu_efi_path_arch)
-    cmd = run_command('test', '-f', join_paths(efi_ldsdir, arch_lds))
+    cmd = run_command('test', '-f', join_paths(efi_crtdir, arch_crt))
   endif
   if cmd.returncode() != 0
-    error('Cannot find @0@'.format(arch_lds))
+    error('Cannot find @0@'.format(arch_crt))
   endif
 endif
 
 message('efi-libdir: "@0@"'.format(efi_libdir))
-message('efi-ldsdir: "@0@"'.format(efi_ldsdir))
+message('efi-crtdir: "@0@"'.format(efi_crtdir))
```

Comment:
Is there a reason for this change? I personally think `ldsdir` is a more helpful name. I also think changing the meson config option is kinda antisocial for distros.

Reply:
I've split out something friendlier in https://github.com/fwupd/fwupd/pull/2921